### ISSUE TITLE
nora: FirebaseError: Firebase: Firebase App named '[DEFAULT]' already exists (app/duplicate-app).

### ISSUE THREAD

**Issue by Wireheadbe** [andrei-tatar/node-red-contrib-smartnora#2]
Adding new nodes gives the following error from time to time when deploying a node-red flow:
```nora: FirebaseError: Firebase: Firebase App named '[DEFAULT]' already exists (app/duplicate-app).```

Subsequently followed by:
```nora: trying again in xx.x sec```

My deployment method in node-red is set to "modified nodes only". Extract after deployment:
```
11 Feb 15:15:00 - [info] Starting modified nodes
11 Feb 15:15:00 - [info] Started modified nodes
11 Feb 15:15:09 - [error] nora: FirebaseError: Firebase: Firebase App named '[DEFAULT]' already exists (app/duplicate-app).
11 Feb 15:15:09 - [warn] nora: trying again in 57.5 sec
```

**Comment by Wireheadbe** [comment]
Restarting the node-red container seems to fix the issue.

**Comment by Wireheadbe** [comment]
double init in connection.ts I presume?
Something like this might workaround it?
```
if (firebase.app.length === 0) {
  firebase.initializeApp(firebaseConfig);
}
```

**Comment by andrei-tatar** [comment]
Should be fixed with the latest release (1.0.0-alpha.2).

**Comment by Wireheadbe** [comment]
No issues after updating
